target_chains/ethereum/contracts/forge-test/Entropy.t.sol

```diff
@@ -877,7 +878,7 @@ contract EntropyTest is Test, EntropyTestUtils, EntropyEvents, EntropyEventsV2 {
                 0
             )
         );
-        vm.expectEmit(false, false, false, true, address(random));
+        vm.expectEmit(true, true, true, false, address(random));
```

Comment:
it turns out that these booleans are which fields of the event to check (4 bools for the 3 topics and data) 🤦 . (This ^ means we check the topics but not the data (which has the callbackGasUsage, which is hard to compute exactly). There is a more explicit test for the data field of this event below.

Also, I'm pretty sure the other calls to expectEmit in this file are doing this wrong, but i'll fix in a separate PR.